README.md

```diff
@@ -212,6 +213,69 @@ See also:
 - [istanbul coverageVariable][istanbul-coverage-variable]
 - [SanboxedModule][sandboxed-module-coverage-variable]
 
+
+### istanbul.enforceThresholds(opt)
+
+Checks coverage against minimum acceptable thresholds. Fails the build if any of the thresholds are not met.
+
+#### opt
+Type: `Object` (optional)
+```js
+{
+  coverageVariable: 'someVariable',
+  thresholds: {
```

Comment:
I would keep it separate, it doesn't add much syntax and it means the thresholds can be easily included from another variable.